homeassistant/components/backup/const.py

```diff
@@ -10,6 +11,16 @@
 if TYPE_CHECKING:
     from .manager import BackupManager
 
+
+class LastBackupState(StrEnum):
+    """Last backup state type."""
+
+    NO_BACKUP = "no_backup"
+    BACKUP_FAILED_LOCATIONS = "last_backup_failed_locations"
+    BACKUP_FAILED = "last_backup_failed"
+    BACKUP_SUCCESSFUL = "last_backup_successful"
```

Comment:
I think this, and the calculation in coordinator, should be moved to `sensor.py` since it's only used by the proposed new sensor

Reply:
sounds absolutely reasonable, but I'll wait until the discussion above (_https://github.com/home-assistant/core/pull/144187#discussion_r2072573104_) is resolved :+1: